Doc/whatsnew/3.13.rst

```diff
@@ -171,6 +171,13 @@ doctest
   :attr:`doctest.TestResults.skipped` attributes.
   (Contributed by Victor Stinner in :gh:`108794`.)
 
+fractions
+---------
+
+* Objects of type :class:`fractions.Fraction` now support integer-style
+  formatting with the ``d`` presentation type. (Contributed by Mark Dickinson
```

Comment:
I afraid that "integer-style formatting" can confuse users which can expect that it always format a fraction as integer. It is my only concern, but I have no better idea. Otherwise LGTM.

Reply:
I'll reword. I agree `integer-style` is misleading. The key point is that minimum width, alignment, fill character, sign options and thousands separator, etc. are supported. It probably wouldn't be terrible just to list all of these.